### DISCUSSION TITLE
에디터에서 제페토계정 로그인이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#666] [Issue Report (Legacy)]
![image](https://user-images.githubusercontent.com/100827088/165256322-d3c6a593-73d0-4881-ad28-bd5c8a9a7189.png)

맞는 계정 정보를 입력했음에도 아래와 같은 로그가 나타나며 로그인이 되지 않습니다.
인터넷은 연결되어 있으며, 총 4개의 계정으로 로그인 시도해보아도 로그인이 되지 않습니다.

제페토 어플, 제페토 스튜디오 웹에서는 정상적으로 로그인이 모두 되는 것을 확인했습니다.

```
Login Failed. username or password is incorrect.
UnityEngine.Debug:LogWarning (object)
ZEPETO.Script.LoginOperation/<Login>d__6:MoveNext ()
Unity.EditorCoroutines.Editor.EditorCoroutine/YieldProcessor:MoveNext (System.Collections.IEnumerator) (at Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:80)
Unity.EditorCoroutines.Editor.EditorCoroutine:ProcessIEnumeratorRecursive (System.Collections.IEnumerator) (at Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:134)
Unity.EditorCoroutines.Editor.EditorCoroutine:MoveNext () (at Library/PackageCache/com.unity.editorcoroutines@1.0.0/Editor/EditorCoroutine.cs:115)
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()
```

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 사용하시는 월드 버전이 1.2.0인지 확인 부탁 드립니다.

**Reply by Cha-SeongHun** [reply]
패키지 버전과 관계없이 로그인이 되지 않네요
타 컴퓨터에서도 동일하고, 다른 개발자분께 여쭤보아도 동일증상이 있다고 합니다

**Reply by Zepeto-TechEvangelist** [reply]
네, 동일 증상 확인되어서 수정 작업 진행하겠습니다. 
불편드려 죄송합니다.
감사합니다.

**Reply by smilejsu82** [reply]
월드 1.2.0 입니다 
전 지금 잘 되고 있는데요...
![image](https://user-images.githubusercontent.com/50188795/165268690-0599b3d2-7687-4367-bf8e-8afaa6bff610.png)

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요. 서버쪽 이슈가 있었습니다. 지금은 수정되어 정상 동작합니다.
감사힙니다.
